release notes/v0.46.0.md

```diff
@@ -8,6 +8,19 @@ k6 `v0.46` is here 🎉! This release includes:
 
 - `#pr`, `<small_break_1>`
 - `#pr`, `<small_break_2>`
+- [browser#868](https://github.com/grafana/xk6-browser/pull/868) Removes support for browser `devtools` option.
```

Comment:
I wonder if this shouldn't be in the browser section below. And here just to be "Browser has a number of breaking changes due to change in vision on how it will be used, full list -> link"

Currently, The whole first page of the release notes is breaking changes ... and they are only in browser.

Anyone not interested in that will just gloss over them, potentially missing something (if we have other ones). 

Additionally reading this list will not really tell users they likely need to rewrite big parts of their scripts(or at least how to start browsers) - this happens in the other section. 

cc @grafana/k6-core

Reply:
Which browser section? You mean the _"Automatic Browser Lifecycle Handling"_ point in _"New Features"_ section? Or you mean adding a dedicated browser section at least for this release?

Reply:
Personally I think providing the full list as it is now it's ok, but maybe we could add an H3 header just before the browser changes inside the _"Breaking changes"_ section with an introduction to the changes and links to the _"Automatic Browser Lifecycle Handling"_ point, which summarizes most of the important changes, similar to what you said I believe.

Our idea is to write a migration guide that will assist users in this transition, but unfortunately this is not yet ready so we can link it directly from the release notes.

Reply:
I would suggest having the following structure:

## Breaking changes

### Automatic Browser Lifecycle Handling

A small introduction (as you already added in the current feature section)

- the list of breaking changes

It sounds a bit unfriendly to get directly the list of breaking changes without any introduction that provides context about so many breaking changes.

Reply:
We can add a link to the migration guide here as well. More info: https://github.com/grafana/k6/pull/3260#discussion_r1290136033

Reply:
See a proposal for reorganization in [9349278](https://github.com/grafana/k6/pull/3260/commits/93492783540421834b0fc09d1ebc1beda27e0499).